Source/Initialization/InjectorMomentum.H

```diff
@@ -76,6 +76,55 @@ private:
     amrex::Real m_ux_th, m_uy_th, m_uz_th;
 };
 
+
+// struct whose getMomentum returns momentum for 1 particle, from random
+// gaussian flux distribution in the specified direction.
+// Along the normal axis, the distribution is v*Gaussian,
+// with the sign set by flux_direction.
+struct InjectorMomentumGaussianFlux
+{
+    InjectorMomentumGaussianFlux (amrex::Real a_ux_m, amrex::Real a_uy_m,
+                                  amrex::Real a_uz_m, amrex::Real a_ux_th,
+                                  amrex::Real a_uy_th, amrex::Real a_uz_th,
+                                  int a_flux_normal_axis, int a_flux_direction) noexcept
+        : m_ux_m(a_ux_m), m_uy_m(a_uy_m), m_uz_m(a_uz_m),
+          m_ux_th(a_ux_th), m_uy_th(a_uy_th), m_uz_th(a_uz_th),
+          m_flux_normal_axis(a_flux_normal_axis),
+          m_flux_direction(a_flux_direction)
+        {}
+
+    AMREX_GPU_HOST_DEVICE
+    amrex::XDim3
+    getMomentum (amrex::Real /*x*/, amrex::Real /*y*/, amrex::Real /*z*/,
+                 amrex::RandomEngine const& engine) const noexcept
+    {
+        using namespace amrex::literals;
+        // Generate the v*Gaussian distribution.
+        amrex::Real const urand = 1._rt - amrex::Random(engine);
+        amrex::Real ur = std::sqrt(2._rt*std::log(1._rt/urand));
+        if (m_flux_direction < 0) ur = -ur;
+
+        // Note that in 2D, with m_flux_normal_axis == 1, uy is out of the plane and so Gaussian and uz is v*Gaussian
+        amrex::Real ux = (m_flux_normal_axis == 0 ? m_ux_th*ur : amrex::RandomNormal(m_ux_m, m_ux_th, engine));
+        amrex::Real uy = (m_flux_normal_axis == 1 && AMREX_SPACEDIM==3 ? m_uy_th*ur : amrex::RandomNormal(m_uy_m, m_uy_th, engine));
+        amrex::Real uz = (m_flux_normal_axis == AMREX_SPACEDIM-1 ? m_uz_th*ur : amrex::RandomNormal(m_uz_m, m_uz_th, engine));
```

Comment:
Tiny:
```suggestion
        amrex::Real const ux = (m_flux_normal_axis == 0 ? m_ux_th*ur : amrex::RandomNormal(m_ux_m, m_ux_th, engine));
        amrex::Real const uy = (m_flux_normal_axis == 1 && AMREX_SPACEDIM==3 ? m_uy_th*ur : amrex::RandomNormal(m_uy_m, m_uy_th, engine));
        amrex::Real const uz = (m_flux_normal_axis == AMREX_SPACEDIM-1 ? m_uz_th*ur : amrex::RandomNormal(m_uz_m, m_uz_th, engine));
```

Reply:
Do we need extra treatment for RZ here?

Reply:
This code is Ok, though I've updated how `flux_normal_axis` is read in and its documentation.